### ISSUE TITLE
What is license of this style?

### ISSUE THREAD

**Issue by buma** [ibikecph/cphbikes#83]
Can I base style for my city on this and use it with credit?

**Comment by emiltin** [comment]
hi buma, sorry for the late answer.

it's licensed under mozilla public license 2.0, and you're welcome to modify it and use it. we would appreciate crediting :-)

however i would strongly recommend  you to check out our new rewritten style: https://github.com/ibikecph/ibikecph-carto. it's available under the same license, but has much better handling of a lot of things, like bike tracks and lanes, layers, way scaling, etc. and is also faster to render. it uses osm2pgsql with LUA preprocessing of the osm data.

you can see the new style used at http://www.ibikecph.dk/#!/x5foz.7gzbj/x5jw7.7gstb.

**Comment by buma** [comment]
Thanks. :+1:  It would be a great idea to add LICENSE file :)

**Comment by emiltin** [comment]
you're right. the new repo has a license file: https://github.com/ibikecph/ibikecph-carto/blob/master/LICENSE.

are you working on a router? we would love to hear if you use or modify the style.

**Comment by buma** [comment]
I'm currently using OpenTripPlanner to create a router for my city. And adding bike lanes map would be a great addition.
OpenCycleMap is very nice but it doesn't show if bike lanes are only on one side or sharrows so firstly I modified default OSM style a bit to show this and then I found some carto styles on Github.
I'll let you know if I'll use it.

I have an idea to create summer bike router (it should find paths where uphill and flat is in the shade based on forest and terrain) based on [Brouter](https://github.com/abrensch/brouter) probably but I don't know if something will come from it.
